app/views/conferences/_details.html.slim

```diff
@@ -1,24 +1,12 @@
 .conference
   dl.dl-horizontal
     dt Location
-    dd = @conference.location
+    dd #{@conference.city}, #{@conference.country}
     dt Date
     dd = @conference.date_range
     dt Twitter
     dd = format_conference_twitter(@conference.twitter)
     dt Homepage
     dd = link_to @conference.url, @conference.url
-    dt Tickets total
```

Comment:
We don't use them in this table. They will be back in one of the upcoming user stories.
I think we shouldn't remove them from the database, at least not for now. I added a note to the project backlog. 👍